test/passing/refs.ml

```diff
@@ -5,6 +5,10 @@ let _ =
   xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx :=
     something very
       looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong ;
+  xxxxxxxxxxxxx xxxxxxxxxxx xxxxxxxxxxxx xxxxxxxxx xxxxxxxxxxxxx xxxxxxxxxx
+    xxxxxxxxxxxxx :=
+    something very
+      looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong ;
```

Comment:
@hhugo what do you think of this one? To me this would be clearer with `:=` on the left margin.

Reply:
I'm with you, the following look better to me. @mbarbin, any feedback ?
```
xxxxxxxxxxxxx xxxxxxxxxxx xxxxxxxxxxxx xxxxxxxxx xxxxxxxxxxxxx xxxxxxxxxx
  xxxxxxxxxxxxx
:=
  something very
    looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong ;
```

Reply:
+1.  I find it is easier to spot the operator `:=` when it is on the left margin.

Reply:
Ok, it sounds like using the previous formatting in case `is_simple` does not hold of the lhs might work.